qa/kerberos/src/main/resources/test.properties

```diff
@@ -0,0 +1,57 @@
+# default settings for Hadoop test
+
+#test.disable.local.es=true
+# ES
+#es.nodes=localhost
+#es.nodes=http://84f306619beadeb16048ae2d4d652ac5.eu-west-1.aws.found.io:9200/
+# by default this is set dynamically based on the port used by the embedded ES instance
+#es.port=9200
+#es.port=9500
+es.batch.size.bytes=1kb
+#es.nodes.client.only=true
+#es.nodes.wan.only=true
+
+# Minimal Security
+es.net.http.auth.user=elastic-admin
+es.net.http.auth.pass=elastic-password
+
+# put pressure on the bulk API
+es.batch.size.entries=3
+es.batch.write.retry.wait=1s
+
+mapred.max.split.size=134217728
+#mapreduce.local.map.tasks.maximum=10
+
+# M&R
+fs.default.name=file:///
+mapred.job.tracker=local
+
+# Locally Deployed MR
+#fs.defaultFS=hdfs://localhost:9000
+#mapreduce.framework.name=yarn
+
+#Hive - only works if the HDFS is setup as well
+hive=local
+# Locally Deployed Hive
+#hive=jdbc:hive2://localhost:10000/
+
+#fs.default.name=hdfs://sandbox:8020
+#mapred.job.tracker=sandbox:50300
+
+#es.net.proxy.http.host=localhost
+#es.net.proxy.http.port=8080
+#es.net.proxy.http.user=test
+#es.net.proxy.http.pass=test
+
+# Pig
+pig.script.info.enabled=false
+
+# Storm
+es.storm.bolt.flush.entries.size=1
+
+# Spark props
+spark.master=local
```

Comment:
re: Local mode?

Reply:
Yeah, it's definitely running in yarn mode (see above).